assets/constants.js

```diff
@@ -15,6 +15,7 @@
 var constants = {
   // Whether to allow custom event reporting to Google Analytics.
   "CAN_SEND_ANALYTICS_EVENTS": false,
+  "IS_CURRENT_RTE_CKEDITOR_4" : true,
```

Comment:
Add a blank line above it, and also add a comment describing the constant.

Reply:
Added PTAL!